### PR TITLE
Touch last seen at on sync api request

### LINKED ISSUE #281: Last seen/playtime are empty
**Describe the bug**

* Last seen appears to just be whenever the account was made
* Playtime seems to always be 0

It doesn't look like there's any code in the MC auth controller to set this last seen time

**To Reproduce**
1. Look up an account that should have playtime in the panel or database

### PR DESCRIPTION
**Pull request by freddyheppell**
## Overview of Changes
* Touch the `last_seen_at` time on `MinecraftAuthTokenController@show` (fixes #281)
* Removes playtime
* Renames `last_seen_at` to `last_synced_at`

## Deployment Requirements
- [ ] Composer update
- [ ] NPM update
- [ ] Database migration
- [ ] .env update
- [ ] Environment upgrade (eg. PHP version)
- [ ] Other (please specify)

### CODE REVIEW

**Comment by freddyheppell** [comment]
It's going to be a bit of a pain to change `last_seen_at` to `last_synced_at` and remove `playtime` whilst the class factory PR is open so I'll hold this until that's merged.

**Comment by freddyheppell** [comment]
Had to split up the migrations into 3 files because you can't rename and modify a column in a single migration, and you can't drop and rename a column in a single migration (although the latter only applies to sqlite)

File: app/Console/Commands/ImportEssentialsBansCommand.php [hunk 1]
```diff
@@ -56,8 +56,6 @@ public function handle()
             if (! $player) {
                 $player = MinecraftPlayer::create([
                     'uuid' => $essBan['uuid'],
-                    'playtime' => 0,
-                    'last_seen_at' => new Carbon($essBan['created']),
```
Comment (andyksaw) [inline app/Console/Commands/ImportEssentialsBansCommand.php:60]: It's probably safe to delete this command entirely, actually 🤔

This was just a quick, dirty way to import some old Essentials plugin bans, but I doubt we'll ever do it this way again

File: app/Entities/Players/Repositories/MinecraftPlayerRepository.php [hunk 2]
```diff
@@ -22,14 +22,12 @@ class MinecraftPlayerRepository extends Repository
      */
     public function store(string $uuid,
                           ?Carbon $lastSeenAt = null,
-                          ?int $accountId = null,
-                          int $playTime = 0): MinecraftPlayer
+                          ?int $accountId = null
+                          ): MinecraftPlayer
```
Comment (andyksaw) [inline app/Entities/Players/Repositories/MinecraftPlayerRepository.php:26]: Whoa this is some funky indentation. I should probably add a linter rule to auto fix this 😅

**Review by andyksaw — APPROVED** [review]
LGTM! 🚀

File: app/Console/Commands/ImportEssentialsBansCommand.php (hunk 1 above)
Comment (andyksaw) [reply app/Console/Commands/ImportEssentialsBansCommand.php:60]: Made an issue for it here: #304 

It's unrelated, so let's merge this

**Pull request merged**